### ISSUE TITLE
Output filename variable

### ISSUE THREAD

**Issue by MicMonen** [gilmarsquinelato/serverless-spa#4]
Can you change the code below:

```
this.webpackConfig.output = {
    path: path.join(process.cwd(), '.spa'),
    filename: '[name].js',
    publicPath: this.webpackConfig.output.publicPath ? this.webpackConfig.output.publicPath : '/'
};
```

To something like this?

```
Object.assign(this.webpackConfig.output, {
      path: path.join(process.cwd(), '.spa'),
});
```

Our filename is different and I think you only want to reassign the path instead of the whole object.

**Comment by gilmarsquinelato** [comment]
Hello @MicMonen you're right, I'll change that!
Thanks for the report.

**Comment by gilmarsquinelato** [comment]
Fixed in v1.2.2

**Comment by MicMonen** [comment]
@gilmarsquinelato Thank you responding quickly and I can confirm its has resolved my problem.
